.github/workflows/ci.yml

```diff
@@ -131,3 +131,22 @@ jobs:
         run: |
           find . -iname '*.c' |\
           xargs complexity --scores --threshold=0 --horrid-threshold=55
+
+  git-secrets:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Checkout awslabs/git-secrets
+        uses: actions/checkout@v2
+        with:
+          repository: awslabs/git-secrets
+          ref: master
```

Comment:
Is this the correct branch?

Reply:
Yes, the git-secrets master branch is used for this coverage.